.ci/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This might be some convention but I feel like this script should be called `setup`, `init` or `install-deps`.

Reply:
I like `install-deps`. Conveys a little more info about what's being installed.